Gemfile

```diff
@@ -1,9 +1,15 @@
 # frozen_string_literal: true
 source 'https://rubygems.org'
 
-gem 'awesome_print'
+ruby '2.3.0'
```

Comment:
Since we're doing this, should we remove the `.ruby-version`?

Reply:
¯\\\_(ツ)_/¯ i want to say that `.ruby-version` still informs `rbenv` on which version to use for local development. But maybe that doesn't matter? Except actually it might because there is stuff that's only available in 2.3+? Except maybe the right answer is to just check for positivity the old fashioned way?

Reply:
I'm happy with any of those. I'm just gonna drop `.positive?` in master

Reply:
Ok, `master` no longer uses `positive?`